source/testnet/references/manage-accounts.rst

```diff
@@ -12,21 +12,21 @@ Accounts
    :backlinks: none
 
 
-An account on the Concordium blockchain consists of two parts. The :ref:`on-chain<glossary-on-chain>`
+An account on the Concordium blockchain is owned by one or more :ref:`credential holders<glossary-credential-holder>` and consists of two parts: The :ref:`on-chain<glossary-on-chain>` part which is publicly visible, and maintained by the bakers, and the :ref:`off-chain<glossary-off-chain>` part.
 part which is publicly visible, and maintained by the bakers, and the
 :ref:`off-chain<glossary-off-chain>` part.
```

Comment:
I think the old and the new sentence were mixed up. Here's what it should look like:

An account on the Concordium blockchain is owned by one or more :ref:`credential holders<glossary-credential-holder>` and consists of two parts: The :ref:`on-chain<glossary-on-chain>` part, which is publicly visible and maintained by the bakers, and the :ref:`off-chain<glossary-off-chain>` part.